assets/app-preact/template/.babelrc

```diff
@@ -1,19 +0,0 @@
-{
-  "plugins": [
-        ["import-pika-web", { "dir": "./dist/web_modules" }],
-        ["@babel/plugin-transform-react-jsx", { "pragma":"h" }],
-        ["@babel/plugin-proposal-object-rest-spread"],
-        ["@babel/plugin-proposal-class-properties", { "loose": true }]
-    ],
-    "presets": [
-        "@babel/react",
-        "@babel/preset-typescript",
-        ["@babel/preset-env", { 
-            "targets": {
-                "esmodules": true
-            },
-            "modules": "false",
-            "useBuiltIns": false
-        }]
-    ]
-}
```

Comment:
Babel config was removed. I think the original Babel setup came from my first cut, but it’s not needed anymore. Of course, someone can always add Babel after the fact, but simpler is better.